docs/locales/zh_CN/LC_MESSAGES/tutorial/FedNdarrayDataLoader.po

```diff
@@ -29,101 +29,211 @@ msgid ""
-msgstr ""
+#: ../../tutorial/FedNdarrayDataLoader.ipynb:685
+msgid ""
+"Once you have obtained the required ``npz`` files, use the previously "
+"mentioned load function to read them into FedNdarray form at. Then, input"
+" them into the model to begin training."
+msgstr "得到需要的 NPZ 之后，使用上面介绍过的 Load 函数读取成 FedNdarray，输入到模型中即可开始训练"
 
-#: ../../tutorial/FedNdarrayDataLoader.ipynb:787
-msgid "小建议"
-msgstr ""
+#: ../../tutorial/FedNdarrayDataLoader.ipynb:790
+msgid "Tips"
+msgstr "小建议"
 
-#: ../../tutorial/FedNdarrayDataLoader.ipynb:798
-msgid "建议将数据转为ndarray类型之后，使用单机版训练引擎进行测试，检查数据格式是否正确匹配模型。然后再使用隐语的联邦框架进行测试，可以提高排查效率."
-msgstr ""
+#: ../../tutorial/FedNdarrayDataLoader.ipynb:801
+msgid ""
+"It is recommended to test the data after converting it to the ndarray "
+"type using a single-machine training engine to verify if the data format "
+"matches the model correctly. Then, you can proceed to test it using the "
+"SecretFlow federated framework, which can improve the efficiency of "
+"troubleshooting."
+msgstr "建议将数据转为 ndarray 类型之后，使用单机版训练引擎进行测试，检查数据格式是否正确匹配模型。然后再使用隐语的联邦框架进行测试，可以提高排查效率."
+
+#: ../../tutorial/FedNdarrayDataLoader.ipynb:802
+msgid ""
+"*Note: When using image datasets, it is important to pay attention to the"
+" dimension ordering.*"
+msgstr "*注意：*在使用图像数据集是要注意维度排列"
+
+#~ msgid "在SecretFlow中加载Numpy数据"
+#~ msgstr ""
+
+#~ msgid "本教程将展示下，怎样在SecretFlow的多方安全环境中加载numpy数据。"
+#~ msgstr ""
+
+#~ msgid "SecretFlow支持\\ ``npy``\\ ，\\ ``npz``\\ 多种格式，接口封装和\\ ``numpy``\\ 保持一致"
+#~ msgstr ""
+
+#~ msgid "环境设置"
+#~ msgstr ""
+
+#~ msgid "接口介绍"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "我们在SecretFlow中提供了类似于\\ ``numpy.load``\\ 的接口\\ "
+#~ "``secretflow.data.ndarray.load``\\ 来将各方数据的\\ ``ndarray``\\"
+#~ " 读取成为一个联邦概念的数据。"
+#~ msgstr ""
+
+#~ msgid "通过secretflow.data.load可以读取多方的numpy文件，构成一个\\ ``FedNdarray``"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "接口介绍：\\ `secretflow.data.load "
+#~ "<https://www.secretflow.org.cn/docs/secretflow/en/source/secretflow.data.html#secretflow.data.ndarray.load>`__"
+#~ msgstr ""
+
+#~ msgid "数据下载与分割"
+#~ msgstr ""
+
+#~ msgid "对数据进行拆分"
+#~ msgstr ""
+
+#~ msgid "分别保存成npz格式文件"
+#~ msgstr ""
+
+#~ msgid "将alice和bob的train_x保存成npy格式。方便后文npy格式读取使用"
+#~ msgstr ""
+
+#~ msgid "加载npz文件"
+#~ msgstr ""
+
+#~ msgid "FedNpz的每一个value是是FedNdarray"
+#~ msgstr ""
+
+#~ msgid "加载npy文件"
+#~ msgstr ""
+
+#~ msgid "加载npy就很简单了，直接调用load接口读取出来就是一个标准的FedNdarray对象"
+#~ msgstr ""
+
+#~ msgid "应该怎样将我已有的数据转成FedNdarray再进行读取？"
+#~ msgstr ""
+
+#~ msgid "那我们怎么样将其他类型的数据转成FedNdarray 数据呢？"
+#~ msgstr ""
+
+#~ msgid "比如我有一个图像数据集，语音数据集，我该怎么样通过FedNdarray将数据传入联邦模型"
+#~ msgstr ""
+
+#~ msgid "我们这里以花卉分类数据集Flower来举个例子"
+#~ msgstr ""
+
+#~ msgid "下载解压后根目录存在 Root = “flower_photos”"
+#~ msgstr ""
+
+#~ msgid "得到需要的NPZ之后，使用上面介绍过的Load函数读取成FedNdarray，输入到模型中即可开始训练"
+#~ msgstr ""
+
+#~ msgid "小建议"
+#~ msgstr ""
+
+#~ msgid "建议将数据转为ndarray类型之后，使用单机版训练引擎进行测试，检查数据格式是否正确匹配模型。然后再使用隐语的联邦框架进行测试，可以提高排查效率."
+#~ msgstr ""
+
+#~ msgid "*注意：在使用图像数据集是要注意维度排列*"
```

Comment:
All strings which start with #~ such as #~ msgid  or #~ msgstr should be removed, because it is redundant.
ref to https://github.com/secretflow/secretflow/blob/main/CONTRIBUTING.md#documentation-update-and-its-multilingual-version